web/templates/stream_settings/new_stream_user.hbs

```diff
@@ -1,7 +1,14 @@
 <tr>
-    <td>
-        <a data-user-id="{{user_id}}" class="view_user_profile" tabindex="0">{{full_name}}</a>
-        {{#if is_current_user}} <span class="my_user_status">{{t "(you)"}}</span>{{/if}}
+    <td class="panel_subscriber_member_list">
+        <span class="pill-container">
+            <a data-user-id="{{user_id}}" class="view_user_profile pill" tabindex="0">
+                {{#if src}}
+                <img class="pill-image" src="{{src}}"/>
+                {{/if}}
+                <span class="pill-value">{{full_name}}</span>
+                {{#if is_current_user}} <span class="my_user_status">{{t "(you)"}}</span>{{/if}}
```

Comment:
In other places, we add the pill in JS by using `user_pill.append_user`. We do so for adding user pills for custom profile fields in user profile modal. I don't think there will be any change in behavior in using any of these methods, but just wanted to mention it once. @timabbott thoughts?

Reply:
should i change the approach then ? and add pills using ```user_pill.append_user``` ?

Reply:
I think we don't need to use the `user_pill` module since this is a bit of a different context, but I would prefer that we at least implement this via passing parameter to the `input_pill` template, possibly with a new one for not including the `exit` icon; we shouldn't be duplicating that HTML 

@sujalshah-bit is that easy to do?

Reply:
I have tried to do with `input_pill` template but UI is breaking 
![image](https://github.com/zulip/zulip/assets/73663475/d122958e-a59e-4263-b598-f85d0f1d3308)